Lesson_34/js/script.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function () {
```

Comment:
устаревший способ. лучше испозовать shorthand
```suggestion
$(function() {
```